doc/user-guides/multi-cluster.md

```diff
@@ -0,0 +1,544 @@
+
+```bash
+kubectl apply -f - <<EOF
+apiVersion: kuadrant.io/v1alpha1
+kind: DNSPolicy
+metadata:
+  name: ${gatewayName}-dnspolicy
+  namespace: ${gatewayNS}
+spec:
+  routingStrategy: loadbalanced
+  loadBalancing:
+    geo: 
+      defaultGeo: US 
+    weighted:
+      defaultWeight: 120 
+  targetRef:
+    name: ${gatewayName}
+    group: gateway.networking.k8s.io
+    kind: Gateway
+EOF
+```    
+Note: the DNSPolicy will leverage the ManagedZone we defined earlier based on the listener hosts defined in the gateway.
+
+Lets check our DNSPolicy has been accepted:
+
+```bash
+kubectl get dnspolicy ${gatewayName}-dnspolicy -n ${gatewayNS} -o=jsonpath='{.status.conditions[?(@.type=="Accepted")].message}'
+```
+
+If you have setup the observability pieces (See installation) and remote write to a Thanos instance, then you should be able to access the Grafana instance and see your deployed gateway and policies in the `platform` engineer dashboard.
+
+## Platform Engineer review
+
+So far we have setup an external gateway, secured it with TLS, Protected all endpoints with a default `DENY ALL` AuthPolicy added a restrictive default RateLimitPolicy and set up ManagedZone and a DNSPolicy to ensure traffic is brought to the gateway for the listener hosts defined. Our gateway is now ready to start accepting traffic.
+
+Once we create a HTTPRoute for our listeners, it will cause the DNSPolicy, Auth and RateLimitPolicy to be `Enforced`. So DNS records will populate auth and rate limiting will be configured and ready to protect requests to that endpoint. 
+
+We can test this by deploying a simple application and connecting it to our gateway.
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/Kuadrant/Kuadrant-operator/main/examples/toystore/toystore.yaml -n ${gatewayNS}
+```
+
+add an `HTTPRoute`:
+
+```bash
+kubectl apply -f - <<EOF
+apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: api
+  namespace: ${gatewayNS}
+spec:
+  parentRefs:
+  - name: ${gatewayName}
+    namespace: ${gatewayNS}
+  hostnames:
+  - "toys.${rootDomain}"
+  rules:
+  - backendRefs:
+    - name: toystore
+      port: 80
+EOF
+```
+
+Let's check our gateway policies are enforced:
+
+```bash
+kubectl get dnspolicy loadbalanced -n ${gatewayNS} -o=jsonpath='{.status.conditions[?(@.type=="Enforced")].message}'
```

Comment:
`loadbalanced` needs to be replaced with `${gatewayName}-dnspolicy` since that is the name of DNSPolicy